src/actions/workspaces.ts

```diff
@@ -178,6 +178,11 @@ export const invalidEditorSessionId = () => ({
   type: actionTypes.INVALID_EDITOR_SESSION_ID
 });
 
+export const finishInvite = (workspaceLocation: WorkspaceLocation) => ({
```

Comment:
Please add tests for the new Action Creators you have added.

Also it would be great if you could move all your session related actions to a separate file! The `workspaces.ts` file is getting too huge.